rites/roman1969/src/locales/fr.ts

```diff
@@ -152,19 +152,24 @@ export const locale: Locale = {
   },
 
   names: {
+    achard_of_saint_victor_bishop: 'Bienheureux Achard, évêque d’Avranches († 1171)', // src: mr_fr_1982_ed2_coutances
     adalbert_of_prague_bishop: 'Saint Adalbert, évêque de Prague et martyr († 997)',
     adalbert_of_prague_bishop_patron_of_poland:
       'Saint Adalbert, évêque de Prague, martyr et patron de la Pologne († 997)',
     adelaide_of_burgundy_empress: 'Sainte Adélaîde, impératrice († 999)',
+    adeline_de_mortain_abbess_and_the_saints_of_savigny:
+      'Sainte Adeline de Mortain († v. 1125) et les saints de Savigny', // src: mr_fr_1982_ed2_coutances
```

Comment:
In the proper missal, it's written like that:

![Screenshot 2024-10-11 at 11 21 45](https://github.com/user-attachments/assets/f04655f5-8014-4e86-9692-bfa658c40310)

Reply:
Thanks, Etienne, for sharing this snippet with us.

Yeah, then it is correct. As I understand it, `de Mortain` is not an actual surname, but it is the location of the Abbey (`Abbaye Blanche de Mortain`). Therefore, we might want to consider changing her English celebration name to `of Mortain`. WDYT? :thinking:

I have noticed that the Missal does not contain the death years in the celebration names. I am not against having them in the French celebration names, however, ATM they are not fully following the official celebration names from the French Roman Missals. I think we should later reconsider the default format for French celebration names, but keep an option to include the death dates (cf #154).

Moreover, `sainte Adeline, sœur de saint Vital`. What should that `de saint Vital` mean? Is it part of her religious name (i.e. `Adeline de saint Vital`)? Or is the religious order she founded? :thinking:

Finally, the text above the introit antiphone lists (presumably) all the saints that are considered the `Savigny Saints`.

Reply:
The dates of death are not included in the names of celebrations in French. I'm just continuing to add them for the sake of consistency with the rest, and to avoid introducing a breaking change for this locale. But indeed, we should be able to introduce an option to add them or not.

> What should that `de saint Vital` mean? Is it part of her religious name (i.e. `Adeline de saint Vital`)? 

No, it simply means that `Saint Adeline` is the sister of `Saint Vitalis` (`vitalis_of_savigny_abbot`).

And actually, `Vitalis` is designated as `of_savigny`. We should probably do the same here for `Adeline`.